index.js

```diff
@@ -89,11 +91,15 @@ async function handleRequest(request) {
 
   const result = await processRequest(request);
 
-  if (result && result.risk > 0.9) {
+  if (result && result.risk > riskThreshold) {
```

Comment:
riskThreshold should be defined just before we use it (line 93)

Reply:
hmm. I'm thinking of the riskThreshold as a configuration variable, so I defined it at the top of the script, so that the dev could easily see it and change it.

Reply:
Then it would be good to add a comment explaining what a riskThreshold is